dashboard/app/views/levels/editors/_all.html.haml

```diff
@@ -10,7 +10,10 @@
 - if !@level.is_a?(BubbleChoice)
   = render partial: 'levels/editors/fields/bubble_choice_sublevel', locals: {f: f}
 
-=render partial: 'levels/editors/fields/callouts', locals: {f: f}
+= render partial: 'levels/editors/fields/callouts', locals: {f: f} unless (@level.is_a?(CurriculumReference) || @level.is_a?(DSLDefined))
+= render partial: 'levels/editors/fields/hide_share_and_remix', locals: {f: f} if (@level.uses_droplet?) || @level.is_a?(Blockly)
+= render partial: 'levels/editors/fields/video', locals: {f: f} unless (@level.is_a?(DSLDefined) || @level.is_a?(CurriculumReference))
```

Comment:
some of this logic for whether to show certain fields is getting a bit haphazard -- but i think this is ok since the main purpose of this chain of PRs is to trim down what's visible, given that the underlying implementation is likely to change substantially as part of the UI redesign.